CHANGELOG.md

```diff
@@ -1,5 +1,108 @@
 # Changelog
 
+## v1.11.0 (03.02.2021)
+
+### Enhancements
+
+#### :gear: Control Request Timeouts ([PR #5692](https://github.com/DevExpress/testcafe/pull/5692))
```

Comment:
Can you actually "control" a value? You can control a feature or behavior. Indeed, you can "control" a value if you mean "persistently check and ensure it's okay". 

But meaning "specify" or "set"... I'm not sure

Reply:
`Set` is good